FrontEnd/src/components/PodPage.js

```diff
@@ -146,6 +149,36 @@ class PodPage extends Component {
 
             });
     }
+
+    deletePod = () => {
```

Comment:
I think deleting a pod should be done on the study page. 
You should have a delete button on each pod card that removes that pod from the list of pods. Then use the same old save button to save changes. This way you can delete several pods at the same time.

Reply:
Nice suggestion but not enough time for a rework of this scale.